lua/gatekeeperc.lua

```diff
@@ -0,0 +1,78 @@
+local ffi = require("ffi")
+
+--Structs
+ffi.cdef[[
+
+struct bp_config {
```

Comment:
I take a look at the "phobos" project here, they define the structs in the way we did: https://github.com/Phobos-Framework/phobos/blob/master/lua/dpdkc.lua#L11

But in the "snabb" project (https://github.com/snabbco/snabb/tree/master/src/core), they seem to define the basic structs and functions in C files, and then wrap these functions in a lua script. Typically, they have a xxx.h, xxx.c, and xxx.lua.